pilot/cmd/pilot-agent/main.go

```diff
@@ -349,7 +349,7 @@ var (
 				Node:                role.ServiceNode(),
 				LogLevel:            proxyLogLevel,
 				ComponentLogLevel:   proxyComponentLogLevel,
-				LogAsJson:           loggingOptions.JSONEncoding,
+				LogAsJSON:           loggingOptions.JSONEncoding,
```

Comment:
This could be a version incompatible change?

Reply:
Istio agent and envoy versions are tightly coupled, we do not have to worry about version incompatibility here